v4/source/channels.yaml

```diff
@@ -1330,3 +1330,64 @@
 
             // GetChannelUnread
             channelUnread, resp := Client.GetChannelUnread(<CHANNELID>, <USERID>)
+
+'/channels/{channel_id}/scheme':
+    put:
+      tags:
+        - channels
+      summary: Set a channel's scheme
+      description: |
+        Set a channel's scheme, to wit sets the scheme_id value of a channel record.
```

Comment:
very Shakespearean ;)

Reply:
I'd prefer if we used more common language here. We have many second/third language english readers using these docs and I'm going to be honest I've never seen this phrase before

Reply:
I'm going to leave it because it's a perfectly good way to say "more specifically".

Reply:
Is that the goal of these docs, though? To me the goal is to be easy to read and understand so I think we should use more common language wherever possible to achieve that goal.

Reply:
I think Joram has a point here. It's quite an obscure phrase (I've never encountered it outside of the theatre/English Literature classes at school), so I think it probably doesn't belong in a technical document that's going to be read by many folks for whom English is a second/third/etc language.

Reply:
I don't agree, sorry.

Reply:
Sorry @mkraft, I agree with JW and GG here. Can we use a more common phrase? Our docs are not localized so we need to be as straight forward as possible.